x-pack/plugins/apm/public/components/shared/kuery_bar/index.tsx

```diff
@@ -101,6 +101,7 @@ export function KueryBar(props: { prepend?: React.ReactNode | string }) {
           selectionStart,
           selectionEnd: selectionStart,
           useTimeRange: true,
+          method: 'terms_agg',
```

Comment:
Can we replace this with the constant as well?

Reply:
I don't think it's really necessary since Typescript guards the type of this param.

<img width="653" alt="Screen Shot 2021-07-09 at 16 46 08" src="https://user-images.githubusercontent.com/55978943/125134339-3eba8880-e0d5-11eb-91c2-399c61d733c6.png">

WDYT?